src/misc/simulation/fscommon/fsdirectories.cpp

```diff
@@ -102,6 +105,19 @@ namespace swift::misc::simulation::fscommon
             if (!d.exists()) { continue; }
             return msfsPackage;
         }
+        // then we look for steam-edition
+        for (QString path : locations)
+        {
+            // there seems to be no constant for the roaming directory, so we have to do some magic
+            // https://doc.qt.io/qt-6/qstandardpaths.html
+            path.replace("Local", "Roaming");
+            const QString msfsPackage =
+                CFileUtils::appendFilePaths(CFileUtils::appendFilePaths(path, "Microsoft Flight Simulator"), "");
```

Comment:
Is the outer ``CFileUtils::appendFilePaths`` required? It only "adds" ``""``.

Reply:
this is indeed unnecessary, I misinterpreted the error message. Will be corrected immediately